wormhole-connect/src/views/v2/Bridge/AmountInput/index.tsx

```diff
@@ -66,7 +66,6 @@ const AmountInput = (props: Props) => {
     route,
   } = useSelector((state: RootState) => state.transferInput);
 
-  const [tokenAmount, setTokenAmount] = useState(amount);
```

Comment:
I've decided to remove the local state var and directly use amount in Redux . That's why I also removed the useEffect below.